Code/Tools/SceneAPI/SceneCore/Events/AssetImportRequest.cpp

```diff
@@ -17,13 +17,103 @@
 #include <SceneAPI/SceneCore/Events/AssetImportRequest.h>
 #include <SceneAPI/SceneCore/Events/SceneSerializationBus.h>
 #include <SceneAPI/SceneCore/Utilities/Reporting.h>
+#include <AzCore/Serialization/Json/JsonUtils.h>
+#include <AzCore/Settings/SettingsRegistryMergeUtils.h>
+#include <AzToolsFramework/Debug/TraceContext.h>
+#include <AzCore/JSON/writer.h>
+#include <AzCore/IO/TextStreamWriters.h>
 
 namespace AZ
 {
     namespace SceneAPI
     {
         namespace Events
         {
+            struct UpdateManifestScope final
+                : public AssetImportRequestReporter
+            {
+                AZ_RTTI(UpdateManifestScope, "{C9DD605E-2A9D-4C0D-A22E-A71B44528420}", AssetImportRequestReporter);
+
+                UpdateManifestScope(AZStd::shared_ptr<Containers::Scene> scene)
+                    : m_scene(scene)
+                {
+                    m_registered = false;
+                    AZ::SettingsRegistry::Get()->Get(m_registered, AZ::SceneAPI::Utilities::Key_AssetProcessorInDebugOutput);
```

Comment:
Nit: Do we need to check whether AZ::SettingsRegistry::Get() returns nullptr here?